### DISCUSSION TITLE
New "Big Sur" icon?

### DISCUSSION THREAD

**Discussion by almeath** [PCemOnMac/PCemV17macOS#5] [Ideas]
I would like to suggest an icon design that matches the new "Big Sur" icon UI .. well it's more iOS really, but I cannot find any tool to help me construct Big Sur icons.

[pcem.icns.zip](https://github.com/PCemOnMac/PCemV17macOS/files/6059775/pcem.icns.zip)

Nothing fancy, but let me know what you think.

**Comment by kyr0** [comment]
Love it. Sorry for being off all the time; got a lot of other things buggin' me these days. How about a PR on the bundler repo? :)

**Comment by kyr0** [comment]
@brunocastello Do you maybe have an idea too? We need you here :)))

**Reply by brunocastello** [reply]
Hi, I will have a look on it friday night... new job is taking up my time =P

**Reply by kyr0** [reply]
Hope it's running well for you :)

**Reply by kyr0** [reply]
Hey @brunocastello :) How's your job going? :)

**Reply by brunocastello** [reply]
> Hey @brunocastello :) How's your job going? :)

Hi! Its going fine, I’ve just replied you on gitter :P
